System/DataModule.cpp

```diff
@@ -353,7 +351,7 @@ namespace RTE {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-	bool DataModule::AddMaterialMapping(int fromID, int toID) {
+	bool DataModule::AddMaterialMapping(unsigned char fromID, unsigned char toID) {
```

Comment:
1. Material IDs are always handled with uchars just like colors. 8bit and stuff you know.
2. If we ever wanted to do that then having a few variables declared as uchars will be the least of our worries because we'd be rewriting the whole system.

Reply:
Okay, if this was just to make it more consistent with everything else I can live with it. It probably wouldn't be bad to do down the line, like was done with moids, but maybe there's less benefit, and it ain't important right now anyway.